tlspuffin/harness/wolfssl/src/put.c

```diff
@@ -198,6 +200,15 @@ static void fill_claim(AGENT agent, struct Claim *claim)
         return;
     }
 
+    // claim.write is used to set the value for outbound in Rust FinishedClaim
+    // only Finished claims with outbound == false (=> write == 0) are used to
+    // check security properties. Here we only want to check security properties
+    // after a ClientFinished has been sent, but we still want Finished with
+    // outbound == true to have some information about PUT internal state if the
+    // handshake doesn't finish
```

Comment:
explain you want this to be able to extract early key material to decrypt messages

Reply:
done